apps/passport-client/src/dispatch.ts

```diff
@@ -180,6 +186,9 @@ async function finishLogin(
 
   // Save PCDs to E2EE storage.
   await saveParticipantPCDs(participant);
+
+  // Ask user to save their sync key
+  update({ modal: "save-sync" });
```

Comment:
can we prompt the user to save their sync key if this modal hasn't popped up before, by i.e. checking and setting a flag in local storage? most people have already created a passport by now and will not have saved their sync key, so only doing this for future users won't actually have that much of an impact.